genesys:
  raw_bag_decoder:
    ros__parameters:
      protocol_version: 'v3.3.4'
      frame_ids:
        navsatfix: 'gnss_link'
        imu: 'imu_link'
        adma: 'adma'
        adma_status: 'adma_status'
        raw_data: 'data_raw'
        odometry_pose_id: 'adma' # header.frame_id (basicly could be a "name" of desired POI)
        odometry_twist_id: 'odometry' # odometry.child_frame_id
      odometry_yaw_offset: 0.0 # in degrees
      topic_pois: # those values can be mixed individually
        navsatfix: 1 # either 1-8 for desired POI or 0 if MRP(Measurement Reference Point) is required
        imu: 1
        velocity: 1
        odometry: 1
      time_mode: 1 # 0 = ADMA INS time / 1 = ROS system time / 2 = external ROS time topic
